### DISCUSSION TITLE
JetBrains

### DISCUSSION THREAD

**Discussion by MarkMerrimanIdea** [community/community#32151] [Other Feature Feedback, Questions, & Ideas]
I am new to Github and I am finding it painful to use. There is supposedly integrated support for it in the IntelliJ Idea IDE. I come from an SVN; world where you just type in the URL, Username, and Password and hit connect and it just works. GIT: Not so much. Can Someone please tell me what am I missing?
I have to be making this hard on myself I just know it!

**Comment by airtower-luna** [comment]
GitHub doesn't allow Git operations with your password as a security measure. You have to [create a personal access token (PAT)](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and use that instead of your password when Git (or your IDE) asks.

Your token needs to have the **repo** scope, plus **workflow** if you use GitHub Actions.
